src/components/RerouteToLogin/index.tsx

```diff
@@ -0,0 +1,65 @@
+import React, {
+  useEffect,
+  useState
+} from 'react';
+
+import {
+  DoubleLeftOutlined
+} from '@ant-design/icons';
+
+import {
+  Button,
+  Tooltip,
+  Flex
+} from 'antd';
+
+import useSHOGunAPIClient from '../../hooks/useSHOGunAPIClient';
+
+export interface RerouteToLoginProps {
+  rerouteMsg: string;
+}
+
+const RerouteToLogin: React.FC<RerouteToLoginProps> = ({
+  rerouteMsg
+}) => {
+
+  const [loginUrl, setLoginUrl] = useState<string>();
+
+  const client = useSHOGunAPIClient();
+  const keycloak = client?.getKeycloak();
+
+  useEffect(() => {
+    const getLoginUrl = async () => {
+      const url = await keycloak?.createLoginUrl();
```

Comment:
Out of curiosity: Shouldn't we have a login-URL already? why does it needs to be created here?

Reply:
How could I get access to that link?

Reply:
Alright, thank you